API.md

```diff
@@ -206,20 +287,19 @@ Start a new job
 
 Parameters
 ```
-    signature: String object containg user signature (signed message)
-    documentId: String object containing the document did
-    consumerAddress: String object containing consumer's ethereum address
-    output: json object that define the output section, i.e. attributes of the compute results
-    algorithmDid: hex str the did of the algorithm to be executed
-    algorithmMeta: json object that define the algorithm attributes and url or raw code
-    jobId: String object containing workflowID (optional)
-    transferTxId: hex str the transaction id (hash) of the token transfer, must match the
-        amount of data tokens expressed in the `initialize` endpoint
-    serviceId: integer identifies a service in the list of services in the DDO document
-    serviceType: type of service that serviceId refers to, must be `compute` in this case
-    datatoken: hex str the ERC20 contract address of the datatoken attached to the documentId (did)
+    signature: String object containg user signature (signed message) (required)
+    consumerAddress: String object containing consumer's ethereum address (required)
+    nonce: Integer, Nonce (required)
+    dataset: Json object containing dataset information
+        dataset.documentId: String, object containing document id (e.g. a DID) (required)
+        dataset.serviceId: String, ID of the service the datatoken is attached to (required)
+        dataset.transferTxId: Hex string, the id of on-chain transaction for approval of datatokens transfer
+            given to the provider's account (required)
+    algorithm: Json object, containing algorithm information
+        algorithm.documentId: Hex string, the did of the algorithm to be executed (optional)
+        algorithm.meta: Json object, defines the algorithm attributes and url or raw code (optional)
 
-    One of `algorithmDid` or `algorithmMeta` is required, `algorithmDid` takes precedence
+    One of `algorithm.documentId` or `algorithm.meta` is required, `algorithm.documentId` takes precedence
```

Comment:
I am not sure which one takes precedence. Can you please double check the code?

Reply:
Fixed, it looks like `algorithm.meta` takes precedence.

https://github.com/oceanprotocol/pod-configuration/blob/0985118b2bf5b413f6bc035d44419fcca2fd583c/src/index.js#L112-L123